src/colorizer/RecordingControls.ts

```diff
@@ -1,146 +1,112 @@
 import ColorizeCanvas from "./ColorizeCanvas";
 
+// TODO: Remove class?
 export default class RecordingControls {
-  private startBtn: HTMLButtonElement;
-  private abortBtn: HTMLButtonElement;
   private recording: boolean;
-  private startAtCurrentFrameChkbx: HTMLInputElement;
 
-  private filePrefixInput: HTMLInputElement;
-  private filePrefixResetBtn: HTMLButtonElement;
-  private useDefaultPrefix: boolean;
-  private defaultPrefix: string;
   private hiddenAnchorEl: HTMLAnchorElement;
```

Comment:
All the HTMLElement member variables that used to be here seem to imply that this wanted to be its own parent component with all those buttons and inputs as children!

Reply:
Yeah, that's part of the planned changes I want to make down the line! I thought there was an issue for this specifically but this is the closest we have (https://github.com/allen-cell-animated/nucmorph-colorizer/issues/30). `RecordingControls` and `TimeControls` should definitely be their own components.

Reply:
..as much as it makes sense to after the new UI...